rancher/install.go

```diff
@@ -63,10 +117,8 @@ func DeployRancherManager(hostname, channel, version, headVersion, ca, proxy str
 		chartRepo = "https://charts.optimus.rancher.io/server-charts/latest"
 	case "prime-optimus-alpha":
 		chartRepo = "https://charts.optimus.rancher.io/server-charts/alpha"
-		flags = append(flags, "--devel")
```

Comment:
I don't get what was a purpose of this, hence removing - both channels `alpha` and `prime-optimus-alpha` are providing versioned charts so `--devel` should be useless here.